packages/angular/migrations.json

```diff
@@ -99,25 +99,25 @@
     },
     "update-angular-config": {
       "cli": "nx",
-      "version": "13.2.0",
+      "version": "13.2.0-bata.1",
```

Comment:
@FrozenPandaz Should this be `beta.2` if we're going to cut a new patch?

Reply:
It would be better but it's alright. In the future, please set the next `beta` versions here.